src/Seq.Input.HealthCheck/HealthCheckInput.cs

```diff
@@ -99,11 +101,17 @@ public void Start(TextWriter inputWriter)
                 BypassHttpCaching,
                 FollowRedirects);
 
+            var checkInterval = TimeSpan.FromSeconds(IntervalSeconds);
+            
+            // offset task starts by 1000th of the check interval to avoid sending simultaneous requests.
+            var taskDelay = checkInterval * ix / 1000;
```

Comment:
Since all we're doing is staggering starts, could we do everything here in one line?

```csharp
await Task.Delay(IntervalSeconds / targetUrls.Length);
```

This would have the effect of evenly spacing out requests across the interval. Over time, it's possible that some checks could end up synchronized, but there should be some jitter introduced by check times that will stir things up a bit.

CC @leighmetzroth

Reply:
(May need to think about cancellation/shut-down, because an hourlong sleep is a problem; also would be nice to avoid the sleep if we only have one URL to check...)

Reply:
I've pushed this suggestion onto my PR along with the suggested changes @liammclennan made in this PR.

I also fixed a minor bug in your suggestion @nblumhardt as it would have just delayed the start of all checks by that amount. Also, correct me if I am wrong, but by adding the `Task.Delay` inside of the health check task taking the cancellation token; it would cancel the delay anyway if the server requests shutdown?